packages/odata-v2/src/uri-conversion/get-expand.spec.ts

```diff
@@ -54,7 +54,8 @@ const testExpandMultiLink = {
 const testNestedExpandLink = {
   expand: testEntityApi.schema.TO_SINGLE_LINK.select(
     testEntitySingleLinkApi.schema.TO_MULTI_LINK.select(
-      testEntityMultiLinkApi.schema.TO_MULTI_LINK
+      // Incorrect type, therefore type assertion to "any", level 2 api does not have navigation properties
```

Comment:
[q] Is this a todo or follow up?

Reply:
No, this is documenting. We need to use type any here to make the ts compiler happy.